src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Carriers/FormTheme/costs-range.html.twig

```diff
@@ -0,0 +1,36 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% block carrier_ranges_costs_zone_range_row %}
+<div class="p-1 col-12 col-md-6 col-lg-3 js-carrier-range-row">
+  {{ form_widget(form.from) }}
+  {{ form_widget(form.to) }}
+  <div class="border p-2">
+    <h3 class="mb-3 js-carrier-range-name">{{ form_widget(form.range) }}</h3>
+    {{ form_label(form.price, null, { 'label_attr': {'class': 'd-block mb-1 text-left' } }) }}
+    {{ form_widget(form.price) }}
+  </div>
+</div>
```

Comment:
You should have a `form_rest(form)` somewhere so that a module can add an extra input if needed, if you don't it will still be output in another unexpected place anyway